library-install-v3.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+set -e
+
+# Necessary to stop pull requests from forks from running.
+if [ "$TRAVIS_SECURE_ENV_VARS" == "true" ]; then
+  npm install
+  npm install --no-save blackbaud/skyux-builder-config#visual-scripts
```

Comment:
We'll need to remove the branch reference once builder config is updated.
https://github.com/blackbaud/skyux-builder-config/pull/22